tests/examples_arguments_syntax/arrow_vector.py

```diff
@@ -0,0 +1,42 @@
+"""
+Arrow Vector
+------------------------------
+This example shows a basic vector plot use point mark ``triangle`` as shape,
+other shape options can be found in :ref:`Point Mark<user-guide-point-marks>`.
+"""
+# category: case studies
```

Comment:
It's probably not a great fit for any of the existing categories. To me, case studies are larger analyses. How about moving it to line charts?
```suggestion
# category: line charts
```
Same change for the other file.